context/app/static/js/pages/SavedList/SavedList.jsx

```diff
@@ -20,22 +21,36 @@ function getListAndItsEntities(savedLists, listTitle) {
 
 function SavedList({ listTitle }) {
   const decodedTitle = decodeURIComponent(listTitle);
+  const [editedListTitle, setEditedListTitle] = useState(false);
+
+  const currentTitle = editedListTitle || decodedTitle;
```

Comment:
Bigger picture: I don't think we should use the title as a key, or put it in the URL (except, down the road, perhaps as a stub which isn't actually needed for URL resolution). So right here, maybe there's just one title, instead of having fallback logic?

Reply:
As we discussed, I'll update everything to use a generated ID instead of the title once we get it deployed for people to test.